portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -66,12 +66,39 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8150 | Test Summary: Page views update in real time"
-	@ignore = "true"
 	@priority = "5"
 	test PageViewsUpdateInRealTime {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Liferay DXP");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentPageViews = selenium.getText("Usage#USAGE_CURRENT_PAGE_VIEWS");
+
+		ACUtils.launchDXP();
+
+		ACUtils.addSiteAndPage();
```

Comment:
@AfonsoHSantos, I believe you don't need to create this other site and page, because you linked the AC on the **Liferay DXP** page so no events will be sent on this other site that was created. What do you think?

Reply:
fixed